src/ledger/LedgerManagerImpl.cpp

```diff
@@ -400,6 +419,17 @@ LedgerManagerImpl::valueExternalized(LedgerCloseData const& ledgerData)
         << ", ops=" << ledgerData.getTxSet()->sizeOp() << ", sv: "
         << stellarValueToString(mApp.getConfig(), ledgerData.getValue()) << "]";
 
+    if (mLastExternalize != std::chrono::steady_clock::time_point())
```

Comment:
Oh I see so the purpose of this metric is to get the overall time spent between a ledger closing and the end of the next consensus round (right before closing the next ledger)?
I think right now we have the submetrics: time spent in nomination and time spent in ballot protocol, so this would allow us to get the number spent overall (basically the sum of those and the time spent before nomination starts).

So should this metric just be the time spent before nomination starts then (as defined in herder, in some cases "nomination starts" is just short circuited by "ballot protocol starts")?

I think that either we have metrics that represent our SLA (`mLedgerAgeClosed` tracks that we are indeed "closing a ledger every 5 seconds"), or metrics that we can easily compare/reason about (so not aggregates themselves).

Reply:
Okay I'll remove this metric because you can calculate the time before nomination starts from ledger age, ballot time, and nomination time. Leaving the slow ledger logging in the new place because it more accurately captures the event "this ledger was slow because SCP was slow".